code/modules/antagonists/ninja/ninja.dm

```diff
@@ -105,7 +105,7 @@
 	return TRUE
 
 /proc/is_ninja(mob/living/M)
-	return M && M.mind && M.mind.has_antag_datum(/datum/antagonist/ninja)
+	return M.mind?.has_antag_datum(/datum/antagonist/ninja)
```

Comment:
M?.mind?.

Reply:
@ike709 suggested this, which is part of cryopod.dm. it serves the same function, but doesn't require &&'s